rosbag2_storage_default_plugins/include/rosbag2_storage_default_plugins/sqlite/sqlite_wrapper.hpp

```diff
@@ -47,6 +47,7 @@ class ROSBAG2_STORAGE_DEFAULT_PLUGINS_PUBLIC SqliteWrapper
 
 private:
   DBPtr db_ptr;
+  std::string uri_;
```

Comment:
When `stdlib` datatypes are part of the class, the complete class can't be `dllexport`ed any longer.
So instead of having `ROSBAG2_STORAGE_DEFAULT_PLUGINS_PUBLIC` attached to the class, it has to be explicitly declared for every public function (constructors included).